app/routes/_gcn.circulars.$circularId/route.tsx

```diff
@@ -10,93 +10,45 @@ import {
   type LoaderFunctionArgs,
   json,
 } from '@remix-run/node'
-import { Link, useLoaderData } from '@remix-run/react'
-import { Button, ButtonGroup, Icon } from '@trussworks/react-uswds'
 
-import { get } from '../_gcn.circulars/circulars.server'
-import { PlainTextBody } from './Body'
-import { FrontMatter } from './FrontMatter'
-import { origin } from '~/lib/env.server'
+import { getUser } from '../_gcn._auth/user.server'
+import { getVersions, moderatorGroup } from '../_gcn.circulars/circulars.server'
+import { feature, origin } from '~/lib/env.server'
 import { getCanonicalUrlHeaders, pickHeaders } from '~/lib/headers.server'
-import { useSearchString } from '~/lib/utils'
 import type { BreadcrumbHandle } from '~/root/Title'
 
 export const handle: BreadcrumbHandle<typeof loader> = {
   breadcrumb({ data }) {
     if (data) {
-      const { circularId, subject } = data
-      return `${circularId}: ${subject}`
+      const { circularId } = data
+      return `${circularId}`
     }
   },
 }
 
-export async function loader({ params: { circularId } }: LoaderFunctionArgs) {
+export async function loader({
+  params: { circularId },
+  request,
+}: LoaderFunctionArgs) {
   if (!circularId)
     throw new Response('circularId must be defined', { status: 400 })
-  const result = await get(parseFloat(circularId))
-  return json(result, {
-    headers: getCanonicalUrlHeaders(
-      new URL(`/circulars/${circularId}`, origin)
-    ),
-  })
+
+  let history: number[] = []
+  if (feature('CIRCULAR_VERSIONS')) {
+    history = await getVersions(parseFloat(circularId))
+    history.reverse() // Sort by descending version
+  }
+  const user = await getUser(request)
+  const userIsModerator = user?.groups.includes(moderatorGroup)
+  return json(
+    { circularId, history, userIsModerator },
+    {
+      headers: getCanonicalUrlHeaders(
+        new URL(`/circulars/${circularId}`, origin)
+      ),
+    }
+  )
```

Comment:
This route doesn't need to set a canonical URL. It's impossible to actually navigate to this route since it is a layout route.

```suggestion
  return { circularId, history, userIsModerator }
```